@types/interfaces.d.ts

```diff
@@ -0,0 +1,7 @@
+export type FormLayoutLabelsPosition = 'aside' | 'top';
+
+export type FormLayoutResponsiveStep = {
+  minWidth?: string | 0;
```

Comment:
I added `0` here as an option since it's explicitly used (as a number) in the documented example snippets. Otherwise it expects a valid CSS lengths so any other number should not work.